ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/PaywallFooter.kt

```diff
@@ -0,0 +1,13 @@
+package com.revenuecat.purchases.ui.revenuecatui
+
+import androidx.compose.runtime.Composable
+
+/**
+ * Composable offering a footer Paywall UI using a bottom sheet. It can be configured from the RevenueCat dashboard.
+ * @param options The options to configure the PaywallFooter if needed.
+ */
+@Suppress("unused")
+@Composable
+fun PaywallFooter(options: PaywallFooterOptions = PaywallFooterOptions.Builder().build()) {
```

Comment:
I renamed the `PaywallBottomSheet` to `PaywallFooter`, since now this won't include the full screen option. It will still be displayed as a bottom sheet, but it made more sense to keep naming the same as in iOS in this case.

Reply:
Additionally, I moved the parameters to use the builder pattern. My main reasons is that it will be much easier to add/deprecate options in the future. Since this is a function, keeping deprecated parameters can potentially grow and adding new parameters would need to keep the order to be usable. 

It will indeed make configuring this sligthly harder... But I think it's acceptable. Lmk if you think otherwise.

Reply:
Should we remove this for now until we work on this mode?

In general I don't think this should be its own type

Reply:
Yeah makes sense! Will remove the footer for now

Reply:
Maybe `PaywallFooterOptions` too?